tests/core/daemon/test_daemon.py

```diff
@@ -438,6 +476,123 @@ async def test_get_routes(mock_lonely_daemon):
     }
 
 
+@pytest.mark.parametrize(
```

Comment:
Do a search for `@datacases`.  Earle and I put it together a few weeks ago.  It is in some ways more verbose.  In others, more clear, better type checked, and has one less layer of indentation iirc.  In some places we want a calculated `.id` attribute and use `@property`.  But that can be a regular attribute as well.

Reply:
Replaced with datacases